playwright/masonry/utils/getServerURL.mjs

```diff
@@ -1,6 +1,6 @@
 // @flow strict
 
-const BASE_DOMAIN = 'http://localhost:8888';
```

Comment:
Ah interesting, this still runs on a local build? Should it run on the Netlify preview deploy instead? That would delay the tests running a bit. Don't know if it matters either way really

Reply:
the playwright tests are configured to run against localhost:3000 currently: https://github.com/pinterest/gestalt/blob/master/playwright/playwright.config.js#L31
 
actually, we can just remove `BASE_DOMAIN` here altogether and just use a relative URL since that will resolve to whatever is configured in the playwright config.

Reply:
ah cool, even better!